I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 45, in script
    show brother with charaenter
  File "game/script.rpy", line 45, in <module>
    show brother with charaenter
NameError: name 'charaenter' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 45, in script
    show brother with charaenter
  File "C:\Users\dasfv\Desktop\renpy\renpy\ast.py", line 1674, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\Users\dasfv\Desktop\renpy\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\dasfv\Desktop\renpy\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 45, in <module>
    show brother with charaenter
NameError: name 'charaenter' is not defined

Windows-10-10.0.19042 AMD64
Ren'Py 8.1.3.23091805
Burnout 0.2
Tue Dec  5 18:23:28 2023
